features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -3422,4 +3422,9 @@
         <ExtensionTypes>{{extension_mgt.extension_types}}</ExtensionTypes>
     </ExtensionManagementService>
 
+    <!-- IdentityDataStore configurations -->
+    <IdentityDataStore>
```

Comment:
let's add a reference to the identity.xml as well. As an example how this configuration will take effect

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/4720/commits/4e255eb73df3e6a6fa68eba205814b5e96cc3cd6